docs/csharp/tour-of-csharp/program-building-blocks.md

```diff
@@ -180,70 +125,67 @@ When an instance method declaration includes a `virtual` modifier, the method is
 
-[!code-csharp[ExpressionUsage](~/samples/snippets/csharp/tour/classes-and-objects/Expressions.cs#L66-L89)]
+:::code language="csharp" source="./snippets/shared/ClassesObjects.cs" ID="UsingExpressions":::
 
 ### Method overloading
 
-Method *overloading* permits multiple methods in the same class to have the same name as long as they have unique signatures. When compiling an invocation of an overloaded method, the compiler uses *overload resolution* to determine the specific method to invoke. Overload resolution finds the one method that best matches the arguments or reports an error if no single best match can be found. The following example shows overload resolution in effect. The comment for each invocation in the `UsageExample` method shows which method is invoked.
+Method *overloading* permits multiple methods in the same class to have the same name as long as they have unique signatures. When compiling an invocation of an overloaded method, the compiler uses *overload resolution* to determine the specific method to invoke. Overload resolution finds the one method that best matches the arguments. If no single best match can be found, an error is reported. The following example shows overload resolution in effect. The comment for each invocation in the `UsageExample` method shows which method is invoked.
 
-[!code-csharp[OverloadUsage](~/samples/snippets/csharp/tour/classes-and-objects/Overloading.cs#L3-L41)]
+:::code language="csharp" source="./snippets/shared/ClassesObjects.cs" ID="Overloading":::
 
-As shown by the example, a particular method can always be selected by explicitly casting the arguments to the exact parameter types and/or explicitly supplying type arguments.
+As shown by the example, a particular method can always be selected by explicitly casting the arguments to the exact parameter types and type arguments.
 
 ## Other function members
 
 Members that contain executable code are collectively known as the *function members* of a class. The preceding section describes methods, which are the primary types of function members. This section describes the other kinds of function members supported by C#: constructors, properties, indexers, events, operators, and finalizers.
 
 The following example shows a generic class called `MyList<T>`, which implements a growable list of objects. The class contains several examples of the most common kinds of function members.
 
-> [!NOTE]
-> This example creates a `MyList` class, which is not the same as the .NET standard <xref:System.Collections.Generic.List%601?displayProperty=nameWithType>. It does illustrate the concepts needed for this tour, but is not a replacement for that class.
-
-[!code-csharp[ListClass](~/samples/snippets/csharp/tour/classes-and-objects/ListBasedExamples.cs#L4-L89)]
+:::code language="csharp" source="./snippets/shared/ClassesObjects.cs" ID="ListExample":::
```

Comment:
I think this is the wrong example to show here, it's showing adding to the list but not using indexers.